sdk/eventhub/event-hubs/test/sender.spec.ts

```diff
@@ -198,6 +198,76 @@ describe("EventHub Sender #RunnableInBrowser", function(): void {
     });
   });
 
+  describe("Create batch", function(): void {
```

Comment:
Can you add a test that makes sure we can still send a batch even after we've reached the maximum message size (e.g. `tryAdd` should return false at some point in the test before sending). Then we can also use the receiver to make sure the correct messages were sent.

Reply:
`should be sent successfully` test case do the exact same thing you mentioned.( in the test case I'm trying to send more messages than the max message limit on the link)

Reply:
Could we add the same test where we set a max size as well? And then also verify the messages we sent are the ones we expect? The current test doesn't do either, and my concern would be that Event Hubs changes their max size limit in the future, and this test stops covering this use case.